platforms/shared/configuration/roles/setup/ambassador/tasks/main.yaml

```diff
@@ -27,17 +27,25 @@
   register: terminal
   when: network['type'] == 'indy'
 
+- name: Get Elastic IP
+  environment:
+    AWS_ACCESS_KEY_ID: "{{ aws.access_key }}"
+    AWS_SECRET_ACCESS_KEY: "{{ aws.secret_key }}"
+  shell: |
+    aws ec2 describe-addresses --public-ips {{ item.services.stewards[0].publicIp }} --region {{ aws.region }} --output json | jq '.Addresses[0].AllocationId'
```

Comment:
What if each organization is in a different K8S cluster and thus should have an own Ambassador with a specific public IP? Will this setup work in that case?